tests/test_all.py

```diff
@@ -259,7 +266,13 @@ def test_zypper_verify_passes(container_per_test: ContainerData) -> None:
     [
         c
         for c in ALL_CONTAINERS
-        if (c not in [INIT_CONTAINER, PCP_CONTAINER] + POSTGRESQL_CONTAINERS)
+        if (
+            c
+            not in (
+                [INIT_CONTAINER, PCP_CONTAINER, KERNEL_MODULE_CONTAINER]
```

Comment:
why is systemd in the kernel container? that seems odd. plus the test's docstring needs to be uppdated

Reply:
Done